assets/styles/style.css

```diff
@@ -409,6 +409,19 @@ section {
   color: #36b37f;
 }
 
+.modal-tags {
+  justify-content: flex-start;
+}
+
+.modal-tags .category {
+  background-color: #ebf0ee;
+  color: #172b4d;
+  font-size: 12px;
+  padding: 8px 12px;
+  border-radius: 8px;
+  line-height: 16px;
+}
+
```

Comment:
- [ ] Great job on implementing the modal window, it looks great and it's functioning properly 💯 However it should be responsive and it should follow the design in Figma both in desktop and in mobile view. Kindly make the necessary changes to fix the mobile version of it 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/202149081-4f1ccdf2-e0e0-4b7e-b4af-71961047ebeb.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/202149120-b88bb4d3-1967-42b4-9499-f9b8460aeec1.png)